docs/merged_objects.md

```diff
@@ -46,6 +46,187 @@ rownames(merged_sce) # matrix row names
+| `singler_results` | If cell typing with `SingleR` was performed, the full result object returned by `SingleR` annotation |
+| `singler_reference` | If cell typing with `SingleR` was performed, the name of the reference dataset used for annotation |
+| `singler_reference_label` | If cell typing with `SingleR` was performed, the name of the label in the reference dataset used for annotation |
+| `singler_reference_source`  | If cell typing with `SingleR` was performed, the source of the reference dataset (default is [`celldex`](http://bioconductor.org/packages/release/data/experiment/html/celldex.html)) |
+| `singler_reference_version`  | If cell typing with `SingleR` was performed, the version of `celldex` used to create the reference dataset source, with periods replaced as dashes (`-`) |
+| `cellassign_predictions` | If cell typing with `CellAssign` was performed, the full matrix of predictions across cells and cell types |
+| `cellassign_reference` | If cell typing with `CellAssign` was performed, the reference name as established by the Data Lab used for cell type annotation |
+| `cellassign_reference_organs` | If cell typing with `CellAssign` was performed, a comma-separated list of organs and/or tissue compartments from which marker genes were obtained to create the reference |
+| `cellassign_reference_source`  | If cell typing with `CellAssign` was performed, the source of the reference dataset (default is [`PanglaoDB`](https://panglaodb.se/)) |
+| `cellassign_reference_version`  | If cell typing with `CellAssign` was performed, the version of the reference dataset source. For references obtained from `PanglaoDB`, the version scheme is a date in ISO8601 format |
+
+
+Unlike for {ref}`individual SingleCellExperiment objects<sce_file_contents:singlecellexperiment sample metadata`, cluster algorithm parameters are not included in these metadata lists because clusters themselves are not included in the merged object.
+
+
+
+### SingleCellExperiment sample metadata
+
+Sample metadata describing each sample included in the merged object is stored in one of two locations, depending on the sample type:
+
+If the project does_not_ contain multiplexed libraries, this information can be found as additional columns in the `colData` slot's `DataFrame`, along with [cell metrics](#singlecellexperiment-cell-metrics).
+
+```r
+colData(merged_sce) # sample metadata for projects without multiplexing
+```
+
+If the project contains multiplexed libraries, this information is stored in the `metadata` slot in the `sample_metadata` field as a `data.frame`:
+
+```r
+metadata(merged_sce)$sample_metadata # sample metadata for proejcts with multiplexed samples
+```
+
+
+Note that, in {ref} `individual library objects<sce_file_contents:singlecellexperiment sample metadata>`, this sample metadata information is instead stored in the `SingleCellExperiment` object's `metadata` slot.
```

Comment:
I think I would move this sentence to be right after line 189 and state that just like merged objects with multiplexed libraries, all individual objects will contain the `sample_metadata` in the `metadata` slot.